Description: Don't use /proc/self/exe as the renderer path. That is never correct for oxide
Author: Chris Coulson <chris.coulson@canonical.com>

Index: oxide/chromium/src/content/common/child_process_host_impl.cc
===================================================================
--- oxide.orig/chromium/src/content/common/child_process_host_impl.cc	2013-08-16 02:06:45.111140339 +0100
+++ oxide/chromium/src/content/common/child_process_host_impl.cc	2013-08-16 02:06:45.107140339 +0100
@@ -91,7 +91,7 @@
   child_path = CommandLine::ForCurrentProcess()->GetSwitchValuePath(
       switches::kBrowserSubprocessPath);
 
-#if defined(OS_LINUX)
+#if 0
   // Use /proc/self/exe rather than our known binary path so updates
   // can't swap out the binary from underneath us.
   // When running under Valgrind, forking /proc/self/exe ends up forking the
